go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -929,3 +936,43 @@ func (bnc *BaseNetworkController) handlesPodIPAllocation() bool {
 	}
 	return true
 }
+
+func (bnc *BaseNetworkController) shouldReleasePod(expectedSwitchName, switchName string, pod *kapi.Pod, podIfAddrs []*net.IPNet, podDesc string) (bool, error) {
+	isStale, err := kubevirt.IsMigratedSourcePodStale(bnc.watchFactory, pod)
```

Comment:
Would it add clarity to enclose this within an `if kubevirt.IsPodLiveMigratable(pod)`?

Reply:
Let's call `IsPodLiveMigratable` inside  `IsMigratedSourcePodStale` is just checking an annotation and it's kind of make sense whatever is not kubevirt is not a stable kubevirt pod.